data/rewards/calculator.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 type RewardYear struct {
-	StartTime   time.Time
-	CloseTime   time.Time
-	Distributed *balance.Amount
+	StartTime     time.Time
+	CloseTime     time.Time
+	Distributed   *balance.Amount
+	TillLastCycle *balance.Amount // distributed till last cycle
```

Comment:
Do you mean the amount that has been distributed up to the last cycle? Not including the current cycle?

Reply:
> Do you mean the amount that has been distributed up to the last cycle? Not including the current cycle?

Yes. Exactly. This is the amount I should use to forecast the rewards for next cycle.